apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Enums/CalculationProgressStatus.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Energinet.DataHub.WebApi.GraphQL.Enums;
+
+public enum CalculationProgressStatus
+{
+    [GraphQLName("pending")]
```

Comment:
Question: Why lower case here, and not in ProgressStep?

Reply:
So it can be used as direct input to `watt-progress-tracker` ([code](https://github.com/Energinet-DataHub/greenforce-frontend/blob/c39edd45add167f0bb2cf58cde2f40c020eb0b1b/libs/watt/src/lib/components/progress-tracker/src/watt-progress-tracker-step.component.ts#L58)). But I'm not sure what the correct approach is here. The options are:

1. Convert this specific enum to lowercase in GraphQL, then pass the value directly (this is the current approach). Simple, but breaks the naming convention of enums.
3. Do NOT convert to lowercase in GraphQL, but then call `toLowerCase()` in the template. Kinda annoying.
4. Do NOT convert to lowercase in GraphQL, but require UPPERCASE in `watt-progress-tracker` input instead. Goes against the convention of our watt components.
5. Do NOT convert to lowercase in GraphQL, but accept both lowercase and UPPERCASE in `watt-progress-tracker`. Adds additional complexity to the component.
6. Convert ALL enums to lowercase in GraphQL. Major refactor required, but keeps things consistent at least.
7. Use some sort of transform middleware in Apollo/codegen that automatically transforms enums to lowercase in the frontend. Adds a lot of complexity and possibly confusion (why is the type of this enum different from the schema?).

I think I just went with 1, since it was the simpler approach. Perhaps if we make the `CalculationProgressStatus` ObjectType more generic (`ProgressStatus` instead) and add a comment that it is a `watt-progress-tracker` status?

Reply:
Maybe the mapping between the enum and watt-progress-tracker should be in the frontend. So the BFF doesnt have to know about the implementation details of watt-progress-tracker?
Maybe watt-progress-tracker can take a transform function that does the custom mapping?